config/crd/bases/policies.kubewarden.io_clusteradmissionpolicies.yaml

```diff
@@ -108,20 +108,20 @@ spec:
                   the API request to be rejected. The default behaviour is "Fail"
                 type: string
               matchPolicy:
-                description: "matchPolicy defines how the \"rules\" list is used to
-                  match incoming requests. Allowed values are \"Exact\" or \"Equivalent\".
-                  \n - Exact: match a request only if it exactly matches a specified
+                description: 'matchPolicy defines how the "rules" list is used to
+                  match incoming requests. Allowed values are "Exact" or "Equivalent".
+                  <ul> <li> Exact: match a request only if it exactly matches a specified
                   rule. For example, if deployments can be modified via apps/v1, apps/v1beta1,
-                  and extensions/v1beta1, but \"rules\" only included `apiGroups:[\"apps\"],
-                  apiVersions:[\"v1\"], resources: [\"deployments\"]`, a request to
-                  apps/v1beta1 or extensions/v1beta1 would not be sent to the webhook.
-                  \n - Equivalent: match a request if modifies a resource listed in
-                  rules, even via another API group or version. For example, if deployments
+                  and extensions/v1beta1, but "rules" only included `apiGroups:["apps"],
+                  apiVersions:["v1"], resources: ["deployments"]`, a request to apps/v1beta1
+                  or extensions/v1beta1 would not be sent to the webhook. </li> <li>
+                  Equivalent: match a request if modifies a resource listed in rules,
+                  even via another API group or version. For example, if deployments
```

Comment:
the CRD are being generated again, should we split this in another PR/commit?

Reply:
Ops, I should not add the CRDs in this PR. Sorry.